### PR TITLE
Update database error 

### PR DESCRIPTION
**Pull request by khainee**

### CODE REVIEW

File: bot/helper/ext_utils/db_handler.py [hunk 1]
```diff
@@ -47,7 +47,7 @@ async def db_load(self):
                 user_data[uid] = row
             LOGGER.info("Users data has been imported from Database")
         # Rss Data
-        if await self.__db.rss[bot_id].find_one():
+        if await self.__db.rss[str(bot_id)].find_one():
             rows = self.__db.rss[bot_id].find({})  # return a dict ==> {_id, link, last_feed, last_name, filters}
```
Comment (a0v0) [inline bot/helper/ext_utils/db_handler.py:51]: This too

Comment (a0v0) [inline bot/helper/ext_utils/db_handler.py:51]: This too

**Comment by a0v0** [comment]
#107

**Comment by khainee** [comment]
Also solved server side sync command error

**Comment by khainee** [comment]
And I think you're forgot to recall bisync from module.So i added it

**Pull request closed without merging**